File: sea-shell-service-spring-web-flux/sea-shell-rest-service/src/main/java/org/jesperancinha/shell/webflux/service/SeaShellOneAdapter.java
package org.jesperancinha.shell.webflux.service;

import org.jesperancinha.shell.webflux.repo.*;

public class SeaShellOneAdapter {

    protected final ShellRepositoryImpl shellRepository;

    protected final ShellPersonRepositoryImpl shellPersonRepository;

    protected final ShellCostumeRepositoryImpl shellCostumeRepository;

    protected final ShellAccountRepositoryImpl shellAccountRepository;

    protected final ShellTopRepositoryImpl shellTopRepository;

    protected final ShellLowerRepositoryImpl shellLowerRepository;


    public SeaShellOneAdapter(ShellRepositoryImpl shellRepository,
                              ShellPersonRepositoryImpl shellPersonRepository,
                              ShellCostumeRepositoryImpl shellCostumeRepository,
                              ShellAccountRepositoryImpl shellAccountRepository,
                              ShellTopRepositoryImpl shellTopRepository,
                              ShellLowerRepositoryImpl shellLowerRepository) {
        this.shellRepository = shellRepository;
        this.shellPersonRepository = shellPersonRepository;
        this.shellCostumeRepository = shellCostumeRepository;
        this.shellAccountRepository = shellAccountRepository;
        this.shellTopRepository = shellTopRepository;
        this.shellLowerRepository = shellLowerRepository;
    }

}

File: sea-shell-service-spring-web-flux/src/main/kotlin/org/jesperancinha/shell/webflux/service/SeaShellOneAdapter.kt
package org.jesperancinha.shell.webflux.service

import org.jesperancinha.shell.webflux.repo.*

open class SeaShellOneAdapter(
    protected val shellRepository: ShellRepositoryImpl,
    protected val shellPersonRepository: ShellPersonRepositoryImpl,
    protected val shellCostumeRepository: ShellCostumeRepositoryImpl,
    protected val shellAccountRepository: ShellAccountRepositoryImpl,
    protected val shellTopRepository: ShellTopRepositoryImpl,
    protected val shellLowerRepository: ShellLowerRepositoryImpl
)